core/utils.ts

```diff
@@ -190,7 +190,14 @@ export function validate(compiledGraph: dataform.ICompiledGraph): dataform.IGrap
     if (!!action.bigquery) {
       if (action.bigquery.partitionBy && action.type === "view") {
         const error = dataform.ValidationError.create({
-          message: `partitionBy is not valid for BigQuery views; it is only valid for tables`,
+          message: `partitionBy/clusterBy are not valid for BigQuery views; they are only valid for tables`,
```

Comment:
Potentially controversial change. My reasoning is that:

* Simplifies error logic, don't have to check for the case with just `partitionBy`, just `clusterBy`, and then the case with the combination of both.

* It's still immediately obvious to the user in any of these scenarios that what they did is not valid for views.

Reply:
SGTM!